modules/apiserversource-kn.adoc

```diff
@@ -113,10 +136,34 @@ Data,
 [id="delete-apiserversource-kn_{context}"]
 == Deleting the ApiServerSource
 
-You can delete the ApiServerSource, trigger, service, service account, cluster role, and cluster binding created in this guide by entering the following `kn` and `oc` commands:
```

Comment:
This is a good introduction for the procedure that follows. I would keep it and add a `.Procedure` title afterwards before the steps.